colorbleed/plugins/maya/publish/extract_yeti_rig.py

```diff
@@ -102,9 +101,10 @@ def process(self, instance):
 
         # Get input_SET members
         input_set = [i for i in instance if i == "input_SET"]
-        members = cmds.sets(input_set[0], query=True)
         # Get all items
-        members = cmds.listRelatives(members, ad=True, fullPath=True)
+        set_members = cmds.sets(input_set[0], query=True)
+        members = cmds.listRelatives(set_members, ad=True, fullPath=True)
```

Comment:
Should we for now also implement the `or []` fallback for list relatives; otherwise this would error if `input_SET` has no members at all.